File: MinIndexSum.cpp
#include<iostream>
#include<bits/stdc++.h>

using namespace std;

int main(){

    string a1[]={"Orange", "Mango", "Lichi", "Apple", "Strawberry"},a2[]={"Strawberry","Orange","Apple"};
    int min=INT_MAX,sum;
    string fruit="";
    for(int i =0;i<5;i++){
        sum=0;
        for(int j = 0;j<3;j++){
            if (a1[i]==a2[j]){
                sum=i+j;
                
            if(sum<min){
            min=sum;
            fruit=a2[j];          
            }
            }

        }
    
        
    }

cout<<fruit;

}

File: minIndexSum.py
l1=["Orange", "Mango", "Lichi", "Apple", "Strawberry"]
l2=["Strawberry","Orange","Apple"]

sum=0
min=1000000
fruit=""
for i in range(0,5):
    for j in range(0,3):
        if(l1[i]==l2[j]):
            
            sum=i+j
            print(sum)

            if(min>sum):
                min=sum
                fruit=l2[j]
                print(fruit," ")

print(fruit)